lib/ClangImporter/ImportType.cpp

```diff
@@ -1445,18 +1445,17 @@ Type ClangImporter::Implementation::importType(clang::QualType type,
                                      resugarNSErrorPointer);
 }
 
-bool ClangImporter::Implementation::shouldImportGlobalAsLet(
-       clang::QualType type)
-{
+VarDecl::Specifier
+ClangImporter::Implementation::specifierForGlobal(clang::QualType type) {
```

Comment:
This is less about the specifier and more about whether we can treat them as stored constant values. I think it's better to keep the old signature here, and turn that into a VarDecl::Specifier at the call site.